### ISSUE TITLE
Support for Python 3.5+

### ISSUE THREAD

**Issue by ronbarrey** [crappycrypto/wincrypto#2]
Are there any plans to add support for Python 3.5+?

**Comment by tekram** [comment]
Any update on this?

Tried it on 3.5.2 and got:

ImportError: No module named 'api'

**Comment by Ana06** [comment]
> ImportError: No module named 'api'

I get the same error with Python 3.4.0 and 3.4.4

**Comment by crappycrypto** [comment]
I tested this with python 3.8 on windows and python 3.7 on linux. Could you be more specific on when the code fails. The example code as included in the readme works fine. For now I'll close this issue. Please re-open or re-submit if you still encounter this problem.

**Comment by nazywam** [comment]
@crappycrypto As far as I can tell the problem was that `pip install wincrypto` defaulted to version 0.2 which produced the following error:
```
>>> from wincrypto import CryptCreateHash
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/michal/work/venv/lib/python3.6/site-packages/wincrypto/__init__.py", line 1, in <module>
    from api import CryptImportKey, CryptEncrypt, CryptDecrypt, CryptExportKey, CryptCreateHash, CryptGetKeyParam, \
ModuleNotFoundError: No module named 'api'
```

This was fixed in 0.2b but beacause it was marked as a release candidate you had to tell python to install this version specifically.

All of that was fixed with the release of 0.3 though, thanks for that!
